phpBB/includes/functions_messenger.php

```diff
@@ -232,7 +232,7 @@ function template($template_file, $template_lang = '', $template_path = '')
 		if ($template_path)
 		{
 			$template_paths = array(
-				$template_path,
+				$template_path . '/' . $template_dir_prefix,
```

Comment:
Shouldn't it be a ternary to avoid an extra `/` if `$template_dir_prefix` is empty?